tc/q_atm.c

```diff
@@ -26,7 +26,7 @@
 #define MAX_HDR_LEN 64
 
 
-static int atm_parse_opt(struct qdisc_util *qu, int argc, char **argv, struct nlmsghdr *n)
+static int atm_parse_opt(struct qdisc_util *qu, int argc, char **argv, struct nlmsghdr *n, const char *dev)
```

Comment:
Shouldn't this patch only be adding `const` in front of device name variable declarations? This function didn't have a `char *dev` argument in previous patches -- why now?

Reply:
Not sure why, but the `dev` parameter was missing in the function definition

Reply:
I'm noticing that the compiler is only compiling certain files if some options are set. That's why we missed this function. For example, for tc/q_atm.c, this is in the Makefile:

    ifeq ($(TC_CONFIG_ATM),y)
        TCSO += q_atm.so
    endif

Therefore, we should check to make sure we have all of these function headers altered by searching through the code. One way is to look for any files that try to assign a function to the `parse_qopt` function pointer member of `struct qdisc_util`:

    $ grep -lr ".parse_qopt"

Same with with the class options:

    $ grep -lr ".parse_copt"